orca-core/src/main/java/com/netflix/spinnaker/orca/conditions/ConditionSupplier.java

```diff
@@ -26,5 +25,5 @@
   /**
    * returns a list of currently unmet conditions.
    */
-  List<Condition> getConditions(Stage stage);
+  List<Condition> getConditions(String cluster, String region, String account);
```

Comment:
As an aside, perhaps this interface should move out of `orca-core` and into `orca-clouddriver` since it's deploy-centric anyhow?

Reply:
+1

Reply:
One thing that just came to mind ... have these tasks been run in any of our environments? May need to do some cleanup to account for the tasks changing package.

Otherwise porting this code to the other package still makes sense to me.

Reply:
Yeah In prestaging from a test pipeline. Clarification question re:cleanup to account, are you referring to cleaning the tasks from the database?

Reply:
Ya, that should be all that's needed. At some point we'll get to fixing the deserialization such that we don't care so much about tasks changing package.